sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1900,14 +1916,35 @@ def test_dataframe_groupby_series(self, agg_type):
       self.skipTest(
           "https://github.com/apache/beam/issues/20967: proxy generation of "
           "DataFrameGroupBy.describe fails in pandas < 1.2")
-    self._run_test(
-        lambda df: df[df.foo > 40].groupby(df.group).agg(agg_type),
-        GROUPBY_DF,
-        check_proxy=False)
-    self._run_test(
-        lambda df: df[df.foo > 40].groupby(df.foo % 3).agg(agg_type),
-        GROUPBY_DF,
-        check_proxy=False)
+
+    def agg(df, group_by, **kwargs):
```

Comment:
nit: i'd try to simplify this. i would try to mimick the anticipated usage.

If Pandas1:
  test all aggregations from new list without extra parameters

otherwise:
  set test `numeric_only`=True . 

I don't think there is much value to check that Pandas 2 will throw error for incorrect usage.

Reply:
Done.